Containerfile

```diff
@@ -14,11 +14,11 @@ FROM registry.fedoraproject.org/fedora:39
 # - https://github.com/osbuild/osbuild/pull/1468
 COPY ./group_osbuild-osbuild-fedora-39.repo /etc/yum.repos.d/
 RUN dnf install -y osbuild osbuild-ostree osbuild-depsolve-dnf && dnf clean all
+
```

Comment:
Any reason for this?

Reply:
Nope, just a relic from some local dev. Will update.